MIGRATING.md

```diff
@@ -0,0 +1,59 @@
+# Okta Java Authentication SDK Migration Guide
+ 
+This SDK uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/). In short, we do not make breaking changes unless the major version changes!
+
+## Migrating from 1.x.x to 2.0.0
+
+Version 2.0.0 of this SDK introduces few breaking changes from previous versions. 
+In addition to few new classes/interfaces, some existing classes/interfaces are no longer backward compatible.
+
+### Package `com.okta.authn.sdk.resource`
+
+- Replaced `com.okta.sdk.resource.user.factor.FactorProfile` interface with `com.okta.authn.sdk.resource.FactorProfile` interface.
+- Replaced `com.okta.sdk.resource.user.factor.FactorProvider` interface with `com.okta.authn.sdk.resource.FactorProvider` interface.
+- Replaced `com.okta.sdk.resource.user.factor.FactorType` interface with `com.okta.authn.sdk.resource.FactorType` interface.
+
+Note: Old interfaces above were pulled in from **okta-sdk-java** Management SDK hitherto. 
+These are now migrated to reside locally within this Authentication SDK. 
+
+- Added below extensions of `com.okta.authn.sdk.resource.FactorProfile` interface:
```

Comment:
This isn't _new_ right? this was how things were in the previous version

Reply:
Yes, this shouldn't need to be mentioned. Will remove it.

Reply:
Will remove this.